presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/SetOperationNodeTranslator.java

```diff
@@ -155,15 +189,54 @@ private AggregationNode computeCounts(UnionNode sourceNode, List<Symbol> origina
                 Optional.empty());
     }
 
+    private WindowNode appendCounts(UnionNode sourceNode, List<Symbol> originalColumns, List<Symbol> markers, List<Symbol> countOutputs, Symbol rowNumberSymbol)
+    {
+        ImmutableMap.Builder<Symbol, WindowNode.Function> functions = ImmutableMap.builder();
+        WindowNode.Frame defaultFrame = new WindowNode.Frame(RANGE, UNBOUNDED_PRECEDING, Optional.empty(), Optional.empty(), CURRENT_ROW, Optional.empty(), Optional.empty(), Optional.empty(), Optional.empty());
```

Comment:
Why `RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT_ROW`? The frame doesn't really matter since there's no ORDER BY, but `ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING` would be more natural and neutral as way to indicate "no frame"

Reply:
OK, I'll change the frame.
I used `RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT_ROW` to indicate "no frame", because this is the default frame that is assumed when frame isn't specified. To me it seems neutral.
I agree that the one you suggested looks better with `count()`